x-pack/solutions/observability/plugins/apm/server/plugin.ts

```diff
@@ -258,20 +258,17 @@ export class APMPlugin
 
     // create .apm-agent-configuration index without blocking start lifecycle
     createApmAgentConfigurationIndex({ client, logger }).catch((e) => {
-      logger.error('Failed to create .apm-agent-configuration index');
-      logger.error(e);
+      logger.warn(`Failed to create .apm-agent-configuration index: ${e.message}`);
```

Comment:
"I downgraded the error to a warning since it will eventually be caught in this try catch. 

 https://github.com/elastic/kibana/blob/0b71a9d698552ae7f45e57179c7f7bb95aae127b/x-pack/solutions/observability/plugins/observability/server/utils/create_or_update_index.ts#L63 
 
However,  I'm tempting to use debug here

Reply:
I think `debug` is ok, we'll show the error message anyways.